### DISCUSSION TITLE
Issue editing my template

### DISCUSSION THREAD

**Discussion by CarlosOC99** [emailjs-com/help#94] [Q&A]
Hello, i'm trying to edit my template to modify some parameters but the platform keeps telling me there is no modifications to save and when I need to do some changes I have to delete my template and create another one, i need some help to fix this if there is a solution.

**Comment by xr0master** [comment]
Can you describe step by step how you edit the content and save it?
Did you try the incognito mode? Perhaps some browser extensions are breaking the site's functionality.
